src/Orleans/Facet/FacetAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+using System;
+
+namespace Orleans
+{
+    /// <summary>
+    /// Marker attribute for facets
+    /// </summary>
+    public abstract class FacetAttribute : Attribute
```

Comment:
Correct, an interface doesn't address all my concerns.
Still, if we decide to keep some marker contract, I agree that an interface is slightly better, since we could make StorageProviderAttribute implement that interface for example, instead of declaring that the attribute is only for facets, just that it is supported by facets. And in the future we can just obsolete the interface if we decide to support any 3rd party attribute.